src/displayapp/screens/WatchFaceNumerals.h

```diff
@@ -0,0 +1,82 @@
+#pragma once
+
+#include <lvgl/lvgl.h>
+#include <chrono>
+#include <cstdint>
+#include <memory>
+#include <displayapp/Controllers.h>
+#include "displayapp/screens/Screen.h"
+#include "components/datetime/DateTimeController.h"
+#include "utility/DirtyValue.h"
+#include "displayapp/apps/Apps.h"
+
+namespace Pinetime {
+  namespace Controllers {
+    class Settings;
+    class NotificationManager;
+  }
+
+  namespace Applications {
+    namespace Screens {
+
+      class WatchFaceNumerals : public Screen {
+      public:
+        WatchFaceNumerals(Controllers::DateTime& dateTimeController,
+                          Controllers::NotificationManager& notificationManager,
+                          Controllers::Settings& settingsController,
+                          Controllers::FS& fs);
+        ~WatchFaceNumerals() override;
+
+        void Refresh() override;
+
+        static bool IsAvailable(Pinetime::Controllers::FS& filesystem);
+
+      private:
+        uint8_t displayedHour = -1;
+        uint8_t displayedMinute = -1;
+
+        uint16_t currentYear = 1970;
+        Controllers::DateTime::Months currentMonth = Pinetime::Controllers::DateTime::Months::Unknown;
+        Controllers::DateTime::Days currentDayOfWeek = Pinetime::Controllers::DateTime::Days::Unknown;
+        uint8_t currentDay = 0;
+        Utility::DirtyValue<bool> notificationState {};
+        Utility::DirtyValue<std::chrono::time_point<std::chrono::system_clock, std::chrono::nanoseconds>> currentDateTime {};
```

Comment:
Instead of storing all of the current day/hour/etc as variables, how about having two time points:
```c++
Utility::DirtyValue<std::chrono::time_point<std::chrono::system_clock, std::chrono::minutes>> currentDateTime;
Utility::DirtyValue<std::chrono::time_point<std::chrono::system_clock, std::chrono::days>> currentDate;
```
Then if the currentDateTime is updated, redraw hours and minutes / AM/PM. Then set `currentDate` to ` std::chrono::time_point_cast<std::chrono::days>(currentDateTime.Get());`, and see if `currentDate` is updated. If so, then redraw the day, day of week etc

Make sure to remove the variables that are unused after this